### DISCUSSION TITLE
<title> تطوير

### DISCUSSION THREAD

**Discussion by harajalyemen** [community/community#39243] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

السلام عليكم ورحمة الله
مساء الخير
اتمنى ان اجد من يساعدني في بعض التعديلات الخاصة بالموقع الخاص بي
وسوف اكون ممتنا للجميع

**Comment by jfullstackdev** [comment]
if you can use online translator, so we can understand one another, that will be much better

**Comment by queenofcorgis** [comment]
Hi @harajalyemen, thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
